db/schema.rb

```diff
@@ -118,6 +118,7 @@
     t.text "additional_information"
     t.jsonb "evidence_prompts", default: []
     t.datetime "evidence_last_run_at"
+    t.string "confirm_dwp_result"
```

Comment:
Is it worth creating a new table called `passporting_benefit` or equivalent and moving this attribute and other benefit attributes to that table?

Reply:
@hiboabd I think how we handle a partner with a passporting benefit and how we present that information to MAAT may help drive out this decision.